dashboards/mintel/flux/flux-metrics-dashboard.json

```diff
@@ -398,18 +398,27 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "flux_daemon_job_duration_seconds_sum{job=\"$instance\"}",
+          "expr": "histogram_quantile(1, sum(rate(flux_daemon_job_duration_seconds_bucket{service=\"$service\"}[10m])) by (le, success))",
           "format": "time_series",
           "intervalFactor": 1,
-          "legendFormat": "{{namespace}} - Total",
+          "legendFormat": "success - {{success}}",
           "refId": "A"
         }
       ],
-      "thresholds": [],
+      "thresholds": [
+        {
+          "value": 120,
```

Comment:
should this be a variable in the config ?

Reply:
yeah but no - it's not yet jsonnet :(